.eslintrc

```diff
@@ -1,20 +1,22 @@
 {
-  "ecmaFeatures": {
-    "jsx": true,
-    "modules": true
-  },
+  "extends": "eslint-config-airbnb",
   "env": {
     "browser": true,
     "mocha": true,
     "node": true
   },
-  "parser": "babel-eslint",
   "rules": {
-    "quotes": [2, "single"],
-    "strict": [2, "never"],
     "react/jsx-uses-react": 2,
     "react/jsx-uses-vars": 2,
-    "react/react-in-jsx-scope": 2
+    "react/react-in-jsx-scope": 2,
+
+    // Temporary disabled for src/index.js
+    // with following by https://github.com/eslint/eslint/issues/2087
+    "block-scoped-var": 0,
+
+    // Temporary disabled for test/*
+    // with following by https://github.com/eslint/eslint/issues/1416
+    "padded-blocks": 0
```

Comment:
I think we have to make sure we are using the correct parser. By looking [here](https://github.com/airbnb/javascript/blob/master/linters/.eslintrc#L2) it should use `babel-eslint`. 
If so, this rule should work.